torchao/quantization/observer.py

```diff
@@ -78,8 +80,13 @@ def get_block_size(
         block_size = list(input_shape)
         block_size[granularity.axis] = 1
         return tuple(block_size)
-    elif isinstance(granularity, PerRow):
+    elif isinstance(granularity, (PerRow, PerToken)):
```

Comment:
maybe create a separate PR to merge these two: https://github.com/pytorch/ao/blob/18dbe875a0ce279739dda06fda656e76845acaac/torchao/quantization/pt2e/observer.py#L1783?

and move it to `torchao/quantization/utils` ?

Reply:
seems like the pt2e one is not used that might, so should be relatively easy to remove